fitwsarah_frontend/src/views/AdminPanelPage/AdminCoachNotes.js

```diff
@@ -91,6 +91,27 @@ function AdminCoachNotes() {
         setEditCoachNoteId(null);
     };
 
+    const handleDeleteClick = async (coachNoteId) => {
+        if (window.confirm('Are you sure you want to delete this coach note?')) {
```

Comment:
Perhaps we would need to localize this sentence?

Reply:
It's the personal trainer panel and I don't think Sarah cares about putting it in french.